controllers/flowcollector_controller.go

```diff
@@ -29,28 +29,29 @@ import (
 // Make sure it always matches config/default/kustomization.yaml:namespace
 // See also https://github.com/operator-framework/operator-lib/issues/74
 const operatorNamespace = "network-observability"
-const cnoNamespace = "openshift-network-operator"
-const ovsFlowsConfigMapName = "ovs-flows-config"
+
+// const cnoNamespace = ""
+// const ovsFlowsConfigMapName = ""
 const finalizerName = "flows.netobserv.io/finalizer"
 
 // FlowCollectorReconciler reconciles a FlowCollector object
 type FlowCollectorReconciler struct {
 	client.Client
 	Scheme              *runtime.Scheme
-	ovsConfigController ovs.FlowsConfigController
+	ovsConfigController *ovs.FlowsConfigController
 	consoleEnabled      bool
 }
 
 func NewFlowCollectorReconciler(client client.Client, scheme *runtime.Scheme) *FlowCollectorReconciler {
 	return &FlowCollectorReconciler{
-		Client: client,
-		Scheme: scheme,
-		ovsConfigController: ovs.NewFlowsConfigController(client,
-			operatorNamespace, cnoNamespace, ovsFlowsConfigMapName),
-		consoleEnabled: false,
+		Client:              client,
+		Scheme:              scheme,
+		ovsConfigController: nil,
+		consoleEnabled:      false,
 	}
 }
 
+//+kubebuilder:rbac:groups=core,resources=namespaces;configmaps,verbs=get;list;watch;create;update;patch;delete
```

Comment:
there's already rbac on group `core` two lines below with the same verbs, i guess you can merge them